client/my-sites/plugins/marketplace/marketplace-test/admin-menu-fetch.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { Button, Card } from '@automattic/components';
+import { useSelector, useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import useSiteMenuItems from 'calypso/my-sites/sidebar-unified/use-site-menu-items';
+import { getIsRequestingAdminMenu } from 'calypso/state/admin-menu/selectors';
+import { requestAdminMenu } from 'calypso/state/admin-menu/actions';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { level1ObjectMap } from 'calypso/my-sites/plugins/marketplace/marketplace-test';
+import CardHeading from 'calypso/components/card-heading';
+
+export default function AdminMenuFetch(): JSX.Element {
```

Comment:
We do not need to review these pages. They are meant as an easy debug panel for developers.